server/src/testFixtures/java/org/elasticsearch/test/TestCluster.java

```diff
@@ -2373,17 +2373,23 @@ public synchronized void assertAfterTest() throws Exception {
     private void assertRequestsFinished() {
         assert Thread.holdsLock(this);
         for (NodeAndClient nodeAndClient : nodes.values()) {
+            DiscoveryNode discoveryNode = getInstanceFromNode(ClusterService.class, nodeAndClient.node()).localNode();
+            TasksService tasksService = getInstanceFromNode(TasksService.class, nodeAndClient.node());
+            String pendingTasks = tasksService.getJobIdsByParticipatingNodes(discoveryNode.getId())
+                .map(uuid -> tasksService.getTask(uuid).toString())
+                .collect(Collectors.joining(",", "[", "]"));
             CircuitBreaker inFlightRequestsBreaker = getInstance(CircuitBreakerService.class, nodeAndClient.name)
                 .getBreaker(CircuitBreaker.IN_FLIGHT_REQUESTS);
             try {
                 // see #ensureEstimatedStats()
                 assertBusy(() -> {
                     // ensure that our size accounting on transport level is reset properly
                     long bytesUsed = inFlightRequestsBreaker.getUsed();
-                    assertThat(bytesUsed)
-                        .as("All incoming requests on node [" + nodeAndClient.name + "] should have finished")
-                        .isEqualTo(0L);
-                });
+                    if (bytesUsed != 0) {
+                        throw new AssertionError("All incoming requests on node [" + nodeAndClient.name + "] should have finished. " +
```

Comment:
is it required to change the assertj style to concrete `throw new AsssertionError`?

Reply:
Don't think it's required, just cherry-picked. Updated it to use assert style